cli/deploycheck/allcomp.go

```diff
@@ -66,6 +264,7 @@ func newHTTPClientFactory() *config.HTTPClientFactory {
 	}
 	return &config.HTTPClientFactory{
 		NewHTTPClient: clientFunc,
+		RetryCount:    3,
```

Comment:
For CLI usage of this retry feature, I think we should default to 5 retries but make it configurable via HZN_HTTP_RETRIES env var.